src/components/GlobalPreferences/Notifications/SubscriptionsForm.js

```diff
@@ -251,6 +251,12 @@ export default function SubscriptionsForm({
           Event
         </Label>
         <DropDown
+          css={`
+            background: ${theme.surfaceInteractive};
+            color: ${theme.surfaceContent};
+            border: 1px solid ${theme.border};
+            font-weight: 400;
```

Comment:
Are these styles specific to this DropDown, or is it something we should update in aragonUI?

Reply:
This was an error on my behalf. I did this (make disabled state white) below to the subscribe button and wrongly thought it should apply to the dropdown too.